### DISCUSSION TITLE
New to GitHub

### DISCUSSION THREAD

**Discussion by AbbasAli09** [community/community#31907] [New to GitHub] [answered]
Hey everyone !!
I am new to GitHub and I want to learn how to use it , so can anyone help me here ? 
Feel free to email me if you can ;-)

**Comment by melvincwng** [comment, marked as answer]
@AbbasAli09 Hey there, welcome to GitHub Community 👋🏻 

To learn the basics of GitHub, you can try out [GitHub Skills](https://skills.github.com/).

Hope this answers your question 😄
